MixItUp.Base/Model/Overlay/OverlayLeaderboardListItemModel.cs

```diff
@@ -320,8 +325,16 @@ private async Task AddLeaderboardItems(IEnumerable<OverlayListIndividualItemMode
                     }
                 }
 
```

Comment:
This logic below, it works when I tested it, but I'm not sure if there is a smarter place/way to do this.

My thought was: this.lastItems was the old list, and items is the new list.  So, if they both had stuff, and the first item is different... then trigger. Right?

Reply:
That's solid logic, it's what I would have done. Might just change the "!=" to "! .Equals()". I don't know if GUIDs are weird with their "==" comparison or not.